packages/core/src/internal/base/button.base.ts

```diff
@@ -80,13 +83,6 @@ export class CdsBaseButton extends LitElement {
     }
   }
 
-  private setupAnchorFocus() {
-    if (this.anchor) {
-      this.anchor.addEventListener('focus', () => this.setAttribute('focused', ''));
-      this.anchor.addEventListener('blur', () => this.removeAttribute('focused'));
-    }
-  }
-
```

Comment:
Not sure but I don't think this was supposed to be removed? That is what allows the button to apply the focus styles when the user tabs to a button that is a link and I think some other use cases as well. On prod it looks broken when focusing not sure about locally if it has been fixed https://clarity.design/storybook/core/?path=/story/components-button-stories--links

It is also what is breaking the commented test here https://github.com/vmware/clarity/pull/4707/files#diff-84e6a67e36b8a8cef08d1aba22c89482R243

Reply:
i don't recall removing it. so it wasn't something that happened recently. but i'll look into that.

Reply:
I dug into this it looks like the broken styles on the prod docs are were fixed in this commit on master https://github.com/vmware/clarity/commit/458b8aaa3592367ad137b328fbe0e290077a63a7